packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -133,6 +133,7 @@ exports.config = {
     [TerraService, {
       /* Use to change the form factor (test viewport) used in the wdio run. */
       ...FORM_FACTOR && { formFactor: FORM_FACTOR },
+      selector: '[data-terra-dev-site-content] *:first-child',
```

Comment:
did we land on providing an override for the selector? i know we talked about making this an env. full-stack testing will ideally have this set to an orion-application selector

Reply:
I'd love to genericize this selector to not reference terra-dev-site, but i'm not sure how to do that without tying this update to terra-dev site in one way or another.

We could patch terra-dev-site v6 to provide both this data element and a generic one, same with terra dev-site v7.

I *think* it's fair to say that terra-functional-testinig is compatible with both terra dev site version 6 and 7 but not 5 or lower.

Thoughts?

Reply:
Proposed selectors
Options A: `[terra-functional-testing-content] *:first-child`
Option B: `[terra-test-content] *:first-child`

Option A links the selector to this package, Option B hedges in case we change package name in the future. 🤷‍♂️ 

Proposed loading selectors
Option A: `[terra-functional-testing-loading]`
Option B: `[terra-test-loading]`

Reply:
Moved the selector key to top level to ease configuration and updated with the name to be `data-terra-test-content`. https://github.com/cerner/terra-toolkit/pull/464/commits/9f3cf680ca8314a43c1bcf131747bfa07ad4b683

Reply:
Added the attributes to dev site:
v6: https://github.com/cerner/terra-dev-site/pull/314
v7: https://github.com/cerner/terra-dev-site/pull/313 (Admittedly theres a lot more going on in this pr than just data attributes).